toxicity/feature_adder.py

```diff
@@ -226,13 +228,46 @@ def add_features(self, train, test):
         return train_x, test_x
 
 
-if __name__ == "__main__":
-    train = pd.read_csv("data/train.csv")
-    test = pd.read_csv("data/test.csv")
+def save_results(df_train, df_test, param, save=True):
+        """
+        This function create the np matrices with the stimated features.
+        In addition (optional) it saves the matrices as csv files/
+
+        Parameters
+        -------------------------
+        df_train, df_test: pd.Dataframes to go through the transformation
+        param: dictionary with the features that will be calculated in the FeatureAdder class
+        Returns
+        --------------------------
+        (train_x, test_x): Matrices containing all features
 
-    da = FeatureAdder(upper_case=True, word_count=True, unique_words_count=True,
-                      letter_count=True, punctuation_count=True, little_case=True,
-                      stopwords=True, question_or_exclamation=True, number_bad_words=True)
+        Example
+        --------------------------
+        >>> param = {'upper_case':True, 'word_count':True, 'unique_words_count':True,
+             'letter_count':True, 'punctuation_count':True, 'little_case':True,
+             'stopwords':True, 'question_or_exclamation':True, 'number_bad_words':True}
+        >>> train, test = save_results(df_train, df_test, param)
+        """
+        train, test = FeatureAdder(**param).add_features(df_train, df_test)
 
-    df_train, df_test = da.add_features(train, test)
-    print(df_train.head(1))
+        if save:
+            unique_name = time.strftime("%Y%m%d-%H%M%S")
+            # Save data frames
+            name = 'data/output/' + 'df_train ' + unique_name+'.csv'
+            print('Saving train file as {}'.format(name))
+            np.savetxt(name, train, delimiter=",")
+            name = 'data/output/' + 'df_test ' + unique_name+'.csv'
+            print('Saving test file as {}'.format(name))
+            np.savetxt(name, test, delimiter=",")
+            print('Files saved')
+        return train, test
+
```

Comment:
Can we add an easy way to load those datasets? Either another method, or (better?) another optional argument called `load=True` along with a `path` to read from. If `load` is false the feature adder runs, if its true then we just load and return the results.